regression/cbmc/nested_label1/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+main.c
+
+^EXIT=10$
+^SIGNAL=0$
+^VERIFICATION FAILED$
+--
+^warning: ignoring
```

Comment:
:pick: In TG we now require new regression tests to provide a comment below the fold explaining what the test is supposed to be checking - it has been very helpful, particular for tests that lack regexes like this one

Reply:
Good point, I've now done so.